apps/src/p5lab/AnimationTab/AnimationTab.jsx

```diff
@@ -86,7 +97,7 @@ const styles = {
   animationsColumn: {
     display: 'flex',
     flexDirection: 'column',
-    minWidth: 190,
+    minWidth: 240,
```

Comment:
this is a small enough change that it might not be necessary, but should we check in with product about this?

Reply:
Checked and got permission from @moneppo and AmyBW! I'm also making it a change that will only affect Sprite Lab (and not Game Lab)